tf.transpose.tfgex

```diff
@@ -0,0 +1,7 @@
+import tensorflow as tf
+
+inp = tf.compat.v1.placeholder(dtype=tf.float32, shape=(2, 3))
+out = tf.transpose(inp)
+
+tfgex_set_inputs(inp)
+tfgex_set_outputs(out)
```

Comment:
It it a bit interesting that a .tflite built from this .tfgex file does not have TransposeOptions... ``)